extension/popup/popup.css

```diff
@@ -307,18 +307,22 @@ a#lexicon {
 
 .feedback-controls .happy-icon {
   mask: url("./images/icon-happy.svg");
+  cursor: pointer;
 }
 
 .feedback-controls .sad-icon {
   mask: url("./images/icon-sad.svg");
+  cursor: pointer;
 }
 
 .feedback-controls .happy-icon:hover {
   mask: url("./images/icon-happy-hover.svg");
+  cursor: pointer;
```

Comment:
I don't think these will be necessary for `:hover`, they should inherit the cursor from the previous rules

Reply:
Okay, I'll change it. Thanks!